scripts/anonymise-data.js

```diff
@@ -0,0 +1,78 @@
+#!/usr/bin/env node
+
+const parse = require('csv-parse')
+const fs = require('fs')
+
+const processArgs = (args) => {
+  const config = {
+    filename: '',
+    noOfRecords: 300
+  }
+  if (typeof args[2] === 'undefined') {
+    console.error('You must pass in an input filename')
+    process.exit(1)
+  }
+  config.filename = args[2]
+  if (typeof args[3] !== 'undefined') {
+    const val = parseInt(args[3])
+    if (Number.isInteger(val)) {
+      config.noOfRecords = val
+    }
+  }
+  return config
+}
+
+const config = processArgs(process.argv)
+let processed = 0
+const SbiReplacementList = ['106599008']
+let fakeSbiCounter = 107000000
+let fakeAgreementCounter = 11111
+let fakeParcelId = 1111222
```

Comment:
For consistency with `fakeSbiCounter` and `fakeAgreementCounter`:
```suggestion
let fakeParcelIdCounter = 1111222
```